Library/DasharoSystemFeaturesUiLib/DasharoSystemFeatures.c

```diff
@@ -996,6 +1042,14 @@ DasharoSystemFeaturesCallback (
           Value->u8 = FixedPcdGet8 (PcdDefaultPowerFailureState);
           break;
         }
+      case OPTION_ROM_STATE_QUESTION_ID:
+        {
+          if (Value == NULL)
+            return EFI_INVALID_PARAMETER;
+
+          Value->u8 = GetOptionRomPolicyDefault ();

```

Comment:
```suggestion
          Value->u8 = FixedPcdGetBool (PcdLoadOptionRoms) ? OPTION_ROM_POLICY_ENABLE_ALL :
                                                                                                            OPTION_ROM_POLICY_DISABLE_ALL;
```

Much easier and more readable what the default is.

Reply:
I used a function because the value appears twice, but fine with conditional expressions.  Updated.

P.S. Missed review here earlier because GitHub didn't subscribe me automatically as I expected, so thanks for reminding.